docs/l3/index.md

```diff
@@ -6,14 +6,43 @@ L3 will demonstrate the value of creating a shared public infrastructure for man
+| attachment  | A file that needs to be communicated over shared infrastructure consistently, but will not affect he validity of a transaction                      | `/v1/attachment`             |
+
+### Transaction types
+
+Transactions communicate via the distributed infrastructure (ledger and file-store) various messages about the above entity types. The ontology of these is as follows:
+
+| transaction type | description                                                                                                                                                                                                                   | route prefix                                                           |
+| ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------- |
+| Demand create    | Communicates a new demand                                                                                                                                                                                                     | `/v1/demandA/{demandAId}/creation`, `/v1/demandB/{demandBId}/creation` |
+| Match propose    | Communicates a suggestion of a match between an order and a capacity                                                                                                                                                          | `/v1/match2/{match2Id}/proposal`                                       |
+| Match Accept     | Communicates the acceptance of a match by either the owner of the order or the capacity. Once Both parties have accepted a match it is allocated                                                                              | `/v1/match2/{match2Id}/accept`                                         |
+| Match reject     | Communicates the rejections of a match by either the owner of the order or the capacity. Match rejection from a single party prevents the match from ever being allocated                                                     | `/v1/match2/{match2Id}/reject`                                         |
+| Match cancel     | Unilaterally cancels a match that has previously been allocated due to unforeseeable circumstances. This can be triggered by either the order or capacity owner and communicates the demand in question no longer exists      | `/v1/match2/{match2Id}/cancellation`                                   |
+| Demand Comment   | Communicates information about a demand that does not affect the validity of other flows. For example this may communicate the location of a demand as part of a fulfilment process. May be called on any demand by any party | `/v1/demandA/{demandAId}/comment`, `/v1/demandB/{demandBId}/comment`   |
```

Comment:
IDK if I missed it, but is 'match2-comment' included somewhere in the docs?

Reply:
I'm guessing not

Reply:
Hang on there's no match2-comment flow. This only appears from a user PoV when fetching matches. I think this is ok